codecov.yaml

```diff
@@ -11,8 +11,19 @@ comment:
   after_n_builds: 1
   show_carryforward_flags: true
 
+github_checks:
+  annotations: true
```

Comment:
@wenqiq Could you help to update this? I think we discussed this and it would be better to set as `false`.

Reply:
Btw, could you also squash the commits? thanks.